web/sites/default/settings.php

```diff
@@ -761,6 +761,9 @@
   'bower_components',
 ];
 
+// Add the newrelic endpoint for the Metrics API
+$settings["new_relic_rpm.metrics.base_url"] = "https://metric-api.newrelic.com/metric/v1";
```

Comment:
Can we just put this in the cloudgov settings? New relic isn't relevant for local since we don't add it to the config.

Reply:
Yeah that could work for environments. But it we ever want to add a new metric (and test it out first locally), we will want to be able to override in the settings here I think. Unless there is a better way to do that

Reply:
Ok it's fine to leave it then!